frontend/app/app.module.ts

```diff
@@ -16,7 +16,6 @@ import { RouterModule } from '@angular/router';
 import { AppComponent } from './app.component';
 
 import {
-    AnalyticsIdConfig,
```

Comment:
Can you delete this class?

Reply:
which line? I don't see any highlighted lines. But if you mean "`if (analyticsId && router && resourceLoader && window.location.hostname !== 'localhost') {`" and beneath. I didn't remove it, I just cut it up in separate functions.

Reply:
That class was only used in the configAnalyticsId function, which returned a AnalyticsConfig class. This functions was used in the providers. I removed that provider because we don't need to provide that to the analytics service. I now provide de UIOptions in the analytics services and initialise the AnalyticsConfig class from there.

I hope this makes sense.

Reply:
Yes, therefore we can delete AnalyticsIdConfig

Reply:
yes, so its okay now? because the import is deleted? 

Or do you mean the whole class that is defined in configurations.ts? Because that class is still getting used.

Reply:
But where do we use it? I think we should get rid of this class.

Reply:
I use it in the analytics.service, but your right, we can get rid of it. It’s a little bit overkill to create a class just to hold a string.

I’ll fix it tomorrow.